### PR TITLE
Fixed an issue where users were not able to resize the data editor window on data output tab if their size goes beyond browser window. #7241

### PR DESCRIPTION
**Pull request by pravesh-sharma**

### CODE REVIEW

File: web/pgadmin/tools/sqleditor/static/js/components/QueryToolDataGrid/Editors.jsx [hunk 1]
```diff
@@ -178,8 +226,8 @@ export function TextEditor({row, column, onRowChange, onClose}) {
 
   const value = row[column.key] ?? '';
   const [localVal, setLocalVal] = React.useState(value);
-  const {getCellElement} = useContext(RowInfoContext);
   const pgAdmin = usePgAdmin();
+  const editorRef = React.useRef(null);
```
Comment (adityatoshniwal) [inline web/pgadmin/tools/sqleditor/static/js/components/QueryToolDataGrid/Editors.jsx:230]: editorRef doesn't looks to be used anywhere. check other places.

**Review by adityatoshniwal — CHANGES_REQUESTED** [review]
Fix feature tests

**Comment by pravesh-sharma** [comment]
> Fix feature tests

Feature tests are passing on my machine will ask to re-run again.

File: web/pgadmin/tools/sqleditor/static/js/components/QueryToolDataGrid/Editors.jsx (hunk 1 above)
Comment (pravesh-sharma) [reply web/pgadmin/tools/sqleditor/static/js/components/QueryToolDataGrid/Editors.jsx:230]: It is being passed to Resizable div and then used there.

Comment (adityatoshniwal) [reply web/pgadmin/tools/sqleditor/static/js/components/QueryToolDataGrid/Editors.jsx]: it is passed to ref prop which is not required. Essentially, forwardRef would also be not required unless you want to use StyledDiv in your JSON editor.

Comment (pravesh-sharma) [reply web/pgadmin/tools/sqleditor/static/js/components/QueryToolDataGrid/Editors.jsx]: Yes I have already removed that.

Comment (adityatoshniwal) [reply web/pgadmin/tools/sqleditor/static/js/components/QueryToolDataGrid/Editors.jsx]: You edited your original comment to which I replied.

Comment (pravesh-sharma) [reply web/pgadmin/tools/sqleditor/static/js/components/QueryToolDataGrid/Editors.jsx]: Yes, after replying I realized forward ref is not required so when I updated the code I removed it.

**Pull request merged**